solrcloudpy/utils.py

```diff
@@ -55,12 +55,20 @@ def make_request(host, path):
                                         params=resparams,
                                         headers=headers, data=body, timeout=self.timeout)
 
+                if len(servers) > 0 and hasattr(r, 'status_code') and r.status_code != 200:
+                    print 'exception: ', getattr(r, 'text')
+                    host = servers.pop(0)
+                    return make_request(host, path)
+
                 return SolrResponse(r)
 
             except ConnectionError as e:
                 print 'exception: ', e
```

Comment:
feel free to delete this line. Maybe using the `logging` module in the future might help here if we want to see the exception